### PR TITLE
migration: Fix image path error

### PR DESCRIPTION
**Pull request by cliping**
nfs_mount_dir is None in copy storage migration. So need to add
 a default image path for copy storage migration.

### CODE REVIEW

**Comment by cliping** [comment]
Before:
VMStartError: VM 'avocado-vt-vm1' failed to start: error: Failed to start domain 'avocado-vt-vm1'error: Cannot access storage file 'jeos-27-x86_64.qcow2' (as uid:107, gid:107): No such file or directory(exit status: 1)

After:
 (1/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.unix_proxy.unix_socket_path_exist.copy_storage_all.p2p: PASS (174.81 s)
 (2/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.unix_proxy.unix_socket_path_exist.copy_storage_all.non_p2p: PASS (174.55 s)
 (3/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.unix_proxy.unix_socket_path_exist.copy_storage_inc.p2p: PASS (174.41 s)
 (4/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.unix_proxy.unix_socket_path_exist.copy_storage_inc.non_p2p: PASS (172.75 s)
 (5/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.unix_proxy.unix_socket_path_non_exist.copy_storage_all.p2p: PASS (125.14 s)
 (6/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.unix_proxy.unix_socket_path_non_exist.copy_storage_all.non_p2p: PASS (123.59 s)
 (7/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.unix_proxy.unix_socket_path_non_exist.copy_storage_inc.p2p: PASS (125.16 s)
 (8/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.unix_proxy.unix_socket_path_non_exist.copy_storage_inc.non_p2p: PASS (124.82 s)

File: provider/migration/base_steps.py [hunk 1]
```diff
@@ -64,15 +64,20 @@ def setup_default(self):
...
 
-        disk_dict = {'source': {'attrs': {'file': os.path.join(
-            self.params.get("nfs_mount_dir"),
-            os.path.basename(
-                self.vm.get_first_disk_devices()['source']))}}}
+        if nfs_mount_dir:
+            disk_dict = {'source': {'attrs': {'file': os.path.join(nfs_mount_dir,
+                         os.path.basename(self.vm.get_first_disk_devices()['source']))}}}
+        else:
+            default_image_path = "/var/lib/avocado/data/avocado-vt/images"
```
Comment (dzhengfy) [inline provider/migration/base_steps.py:77]: Is it better to use` os.path.join(data_dir.get_data_dir(), 'images')`

**Review by dzhengfy — COMMENTED** [review]
Others LGTM

File: provider/migration/base_steps.py (hunk 1 above)
Comment (cliping) [reply provider/migration/base_steps.py]: Updated, thanks.

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**